modules/Settings/Vtiger/models/Menu.php

```diff
@@ -55,9 +55,7 @@ public function getUrl()
 	{
 		$url = $this->get('linkto');
 		$url = App\Purifier::decodeHtml($url);
-		$url .= '&block=' . $this->getId();
-
-		return $url;
+		return $url . '&block=' . $this->getId();
```

Comment:
return  App\Purifier::decodeHtml($this->get('linkto')) . '&block=' . $this->getId();